### PR TITLE
Fixed Box-shadow on serviceMeshBooksPage

### LINKED ISSUE #1863: [Learn] Rectify Box-Shadow on Service Mesh Books Page
**Current Behavior**

Box-shadow around book cards is intersecting other cards on [Service Mesh Books Page](https://layer5.io/learn/service-mesh-books)

---

**Desired Situation**

Rectify the Box-shadow property on book cards.

---
**Screenshots**

Screenshot_2021-05-18 Service Mesh Books

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by nandirishav**
**Description**

This PR fixes #1863

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by chinmaym07** [comment]
Hey @nandirishav 👋🏻 
Please run these commands mentioned here in your terminal to sign-off your commits
https://github.com/layer5io/layer5/pull/1872/checks?check_run_id=2631566325
& Please mention the issue number like `This PR fixes #issue-number` .
In your case it will be like this `This PR fixes #1863` as it will help to close the issue directly when the PR is merged.

**Comment by nandirishav** [comment]
I have edited the PR fix statement with the mentioned issue number.
I have also signed off the commits as directed .

**Comment by l5io** [comment]
🚀 Preview for commit 924c88f2feb575bb3cfa4295f0ee9a1a6ed0ef13 at: https://60a69f0204ac6e00bea9b6c2--layer5ng.netlify.app

File: src/sections/Learn/Books-grid/BooksGrid.style.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ export const BooksPageWrapper = styled.div`
         }
         .book_image{
             z-index: 1;
-            box-shadow: 0px 1px 52px 8px rgba(0,0,0,0.3);
+            // box-shadow: 0px 1px 52px 8px rgba(0,0,0,0.3);
```
Comment (Nikhil-Ladha) [inline src/sections/Learn/Books-grid/BooksGrid.style.js:36]: Let's remove, instead of commenting it.

**Review by Nikhil-Ladha — APPROVED** [review]

**Comment by Nikhil-Ladha** [comment]
The sign-off is still missing on the commit. Could you please follow the steps [here]() to fix it?

**Comment by l5io** [comment]
🚀 Preview for commit 68515646ef7beffa0b1d3ab21f7b3ca18972e1b2 at: https://60a8ff2e9972af2014367863--layer5ng.netlify.app

**Comment by nandirishav** [comment]
Screenshot 2021-05-22 183032

facing this error .

**Comment by Nikhil-Ladha** [comment]
Do a git commit -s -a and do git push --force again and see what happens.

**Comment by nandirishav** [comment]
after running this command git commit -s -a 
error

**Comment by Nikhil-Ladha** [comment]
Did you do a git pull on that branch, or something of that sort?

**Comment by nandirishav** [comment]
No I didn't do git pull

**Comment by nandirishav** [comment]
I removed the comment through the git editor after going to that branch and commited it .
Then the signoff thing came up .

**Comment by Nikhil-Ladha** [comment]
Okay, let's not trouble you more with this. We can ignore this for this PR. But please do remember to sign-off your commits in future yo avoid these issues 🙂

**Pull request merged**